coq_tools/find_bug.py

```diff
@@ -2237,20 +2237,29 @@ def make_make_coqc(coqc_prog, **kwargs):
                 )
                 coq_theories_path = os.path.join(coq_lib_path, "theories")
                 coq_user_contrib_path = os.path.join(os.path.join(coq_lib_path, "user-contrib"), "Stdlib")
+                coqpath_paths = os.environ.get('COQPATH', '').replace(';', ':').split(':')
```

Comment:
This will break if we have absolute paths on Windows, starting with `C:/`.  Let's instead do
```suggestion
                coqpath_paths = os.environ.get('COQPATH', '').split(os.pathsep)
```

Reply:
Indeed, thanks!